spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -168,6 +168,8 @@ If the container is unable to resolve the parameterized type of the event object
 The method `fireAsync()` may be called with a specific `Executor` object to be used to invoke observers method.
 The container should provide a default `Executor` when `fireAsync()` is called without specifying one.
```

Comment:
+1 for MUST. I think async events must always work even if an `Executor` is not specified. Although the CDI spec does not use key words from [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) we should probably follow these rules where possible.